packages/file/src/types/systemd/resolved.coffee.md

```diff
@@ -0,0 +1,102 @@
+
+# `resolved.coffee.md`
+
+## Example
+
+Overwrite `/usr/lib/systemd/resolved.conf.d/10_resolved.conf` in `/mnt` to set
+a list of fallback dns servers by using an array and set ReadEtcHosts to true.
+
+```javascript
+require("nikita").file.types.systemd.resolved({
+  target: "/etc/systemd/resolved.conf",
+  rootdir: "/mnt",
+  content:
+    FallbackDNS: ["1.1.1.1", "9.9.9.10", "8.8.8.8", "2606:4700:4700::1111"]
+    ReadEtcHosts: true
+})
+```
+
+Write to the default target file (`/etc/systemd/resolved.conf`). Set a single
+DNS server using a string and also modify the value of DNSSEC.  Note: with
+`merge` set to true, this wont overwrite the target file, only specified values
+will be updated.
+
+```javascript
+require("nikita").file.types.systemd.resolved({
+  content:
+    DNS: "ns0.fdn.fr"
+    DNSSEC: "allow-downgrade"
+  merge: true
+})
+```
+
+## Schema
+
+    schema =
+      type: 'object'
+      properties:
+        'rootdir':
+          type: 'string'
+          description: """
+          Path to the mount point corresponding to the root directory, optional.
+          """
+        'reload':
+          type: 'boolean', default: null
+          description: """
+          Defaults to true. If set to true the following command will be
+          executed `systemctl daemon-reload && systemctl restart
+          systemd-resolved` after having wrote the configuration file.
+          """
+        'target':
+          type: 'string', default: '/usr/lib/systemd/resolved.conf.d/resolved.conf'
+          description: """
+          File to write.
+          """
+
+This action uses `file.ini` internally, therefore it honors all
+arguments it provides. `backup` is true by default and `separator` is
+overridden by "=".
```

Comment:
`backup` property in `@file.ini` is one of the type: `string` or `function`. So, we cannot pass `true` as a value. What value can we pass instead?